hw4/03-form.js

```diff
@@ -1,16 +1,59 @@
 const express = require('express');
+const path = require('path');
 const app = express();
 const port = process.env.PORT || 5001;
 
 // Use middleware static() to serve all static files in the given folder
 app.use(express.static('public'));
+app.use('/index', express.static('public/files'));
 
 // Use middleware urlencoded() to parse an incoming request with a urlencoded payload and return an objectß
-app.use(express.urlencoded({ extended: false }));
+
+app.use(
+  express.urlencoded({
+    extended: false,
+  })
+);
+
+const postHTML = `<html><head></head><body>
+  <form method='post' action='/submit' accept-charset=utf-8>
+  <label for="name">Name: </label>
+  <input type="text" name="name" id="name"><br />
+  <label for="email">Email: </label>
+  <input type="text" name="email" id="email"><br />
+  <label for="textInput">Submit your message</label>
+  <textarea type="text" name="textInput"id="textInput"></textarea><br />
+  <label for=newsLetterInput>Sign up for the newsletter</label>
+  <input type=checkbox name="newsLetterInpu" id=newsLetterInput><br />
+  <input type='submit'>
+  </form></body></html>`;
+
```

Comment:
Just one question if you are serving a static html file do we need postHTML in the code? rest all looks good
and to the point